.circleci/tests/helm_repository_suite_test.go

```diff
@@ -0,0 +1,140 @@
+package tests
+
+import (
+	"flag"
+	"maps"
+	"os"
+	"path"
+	"regexp"
+	"slices"
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/helm"
+	"github.com/gruntwork-io/terratest/modules/k8s"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+	"helm.sh/helm/v3/pkg/chartutil"
+)
+
+const (
+	chartsRootDir = "../../."
+)
+
+var update = flag.Bool("update-golden", false, "update golden test output files")
+
+type helmRepoSuite struct {
+	suite.Suite
+}
+
+func TestHelmRepository(t *testing.T) {
+	suite.Run(t, new(helmRepoSuite))
+}
+
+func (s *helmRepoSuite) SetupSuite() {
+	// Run the helm repo add command
+	// helm repo add wiz-sec https://wiz-sec.github.io/charts
+	if _, err := helm.RunHelmCommandAndGetOutputE(s.T(), &helm.Options{}, "repo", "add", "wiz-chart-test", "https://wiz-sec.github.io/charts"); err != nil {
```

Comment:
Can’t we do it with the native Go library?
 I don't like that we run cli command, I am not sure it is safe against command injection and fun like that :/